packages/composite-checkout-wpcom/src/components/wp-checkout.js

```diff
@@ -94,10 +96,11 @@ export default function WPCheckout( {
 					countriesList={ countriesList }
 					PhoneInput={ PhoneInput }
 					StateSelect={ StateSelect }
+					renderDomainContactFields={ renderDomainContactFields }
 				/>
 			),
 			completeStepContent: <WPContactForm summary isComplete={ true } isActive={ false } />,
-			isCompleteCallback: () => isFormComplete( contactInfo, itemsWithTax ),
+			isCompleteCallback: () => isCompleteAndValid( contactInfo ),
 			isEditableCallback: () => isFormEditable( contactInfo, itemsWithTax ),
```

Comment:
The second argument for `isFormEditable` was removed so you can remove it here (and does that mean that `itemsWithTax` is not used anywhere?)